Examples/Along_Isobath_Average.ipynb

```diff
@@ -0,0 +1,1700 @@
+{
```

Comment:
<p>this cell looks like the same as the cell above?</p><p>then why not making it a function and just call it twice with different inputs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/416/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>